impyute/dataset/corrupt.py

```diff
@@ -43,7 +43,14 @@ def mcar(self):
 
     def mar(self):
         """ Overwrites values with MAR placed NaN's """
-        pass
+        output = self.data.copy()
+        nrow, ncol = self.data.shape
+        for i in range(0, nrow):
```

Comment:
This isn't a blocker, since I guess it works but two nested `for` loops is pretttttty messy, You can achieve the same thing if you put the percentile & assign stuff in a separate function and do a [`numpy.apply_along_axis`](https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.apply_along_axis.html)

```
In [4]: np.apply_along_axis(lambda x: x+1, -1, np.array([[1, 2, 3], [4, 5, 6]]))
Out[4]: 
array([[2, 3, 4],
       [5, 6, 7]])
```